tests/test_simulator.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+
+from unittest.mock import Mock, patch
+
+from roulette import Simulator, Game, Martingale, Table, Wheel
+
+
+class TestSimulator(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.martingale = Martingale(self.table)
+        self.wheel = Wheel()
+        self.game = Game(self.wheel, self.table)
```

Comment:
all these should be local variable. only simulator is needed as attribute in test methods